components/brave_wallet/browser/internal/hd_key.cc

```diff
@@ -34,9 +36,7 @@ namespace {
 constexpr char kMasterSecret[] = "Bitcoin seed";
```

Comment:
```suggestion
// a constant defined by BIP-32 to be the key for HMAC.
// This is safe because the entropy of the derived key comes from the seed phrase
constexpr char kMasterSecret[] = "Bitcoin seed";
```

Reply:
@supermassive can we merge this suggestion in even though it wasn't relevant to your PR?

I came across the name of this constant and was confused about it's purpose so figured this might be helpful even though it's not directly relevant to the PR.

Reply:
Thanks! But will comment in next related PR.
Want to merge this one now.